python/python-quiz.md

```diff
@@ -90,30 +89,7 @@ def sum(a, b):
     """
     return a + b
 ```
-- 
-```
-def sum(a, b):
-    """
-    # >>> sum(4, 3)
-    # 7   
-    
-    # >>> sum(-4, 5)
-    # 1
-    """
-    return a + b
-```
-- 
-```
-def sum(a, b):
-    ###
-    >>> sum(4, 3)
-    7   
-    
-    >>> sum(-4, 5)
-    1
-    ###
-    return a + b
-```
```

Comment:
can you please revert back this piece of code, it is still valid testing task
and it is not a duplication haha

Reply:
@all-contributors please add @monkrus for infrastructure, tests and code